packages/dual-channel/dev/client-entry.js

```diff
@@ -0,0 +1,19 @@
+import 'babel-polyfill'
```

Comment:
also already injected via `babel-loader` with root babel config?

Reply:
Yes, thanks for reminding.